src/fides/api/schemas/connection_configuration/connection_secrets_dynamic_erasure_email.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional, Dict, Any
+
+from fideslang.models import FidesDatasetReference
+from pydantic import BaseModel, EmailStr, Extra, root_validator
+from pydantic.fields import FieldInfo
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets_email import (
+    EmailSchema,
+)
+
+
+class DynamicErasureEmailSchema(EmailSchema):
+    """Schema to validate the secrets needed for a dynamic erasure email connector"""
+
+    third_party_vendor_name: str
+    recipient_email_address: FidesDatasetReference = FieldInfo(
```

Comment:
maybe we should rename this to `recipient_email_addres_field` or something, cause it's not an email address anymore